lib/sign_in/widgets/form.dart

```diff
@@ -70,51 +69,17 @@ class _SignInFormState extends State<SignInForm> {
           ),
           SizedBox(height: 32),
           ConstrainedBox(
-            constraints: BoxConstraints(
-              maxHeight: mediaQuery.size.height -
-                  400 -
-                  mediaQuery.padding.bottom -
-                  mediaQuery.padding.top,
-            ),
-            child: WebView(
-                javascriptMode: JavascriptMode.unrestricted,
-                onWebViewCreated: (controller) => this.controller = controller,
-                onPageFinished: (url) async {
-              javascriptMode: JavascriptMode.unrestricted,
-              onWebViewCreated: (controller) => this.controller = controller,
-              onPageFinished: (url) async {
-                final firstPathSegment = Uri.parse(url).pathSegments.first;
-                if (firstPathSegment == 'login') {
-                  // The JavaScript is meant to isolate the login section.
-                  // Hopefully there will be an option to only get that in a
-                  // non-hacky way in the future.
-                  await controller.evaluateJavascript('''
-                  var node = document.getElementById('loginarea');
-                  var html = document.getElementsByTagName('html')[0];
-                  html.removeChild(html.childNodes[2]);
-                  html.appendChild(node);
-                  document.getElementsByTagName('h2')[0].innerHTML = ''
-                  ''');
-                } else if (firstPathSegment == 'dashboard') {
-                  final cookies =
-                      await controller.evaluateJavascript('document.cookie');
-                  // Yes, this is not elegant. You may complain about it when
-                  // there is a nice way to get a single cookie via JavaScript.
-                  final jwt = cookies
-                      .split('; ')
-                      .firstWhere((element) => element.startsWith('"jwt='))
-                      .replaceAll('"', '')
-                      .substring(4);
-
-                  await services.storage.token.setValue(jwt);
-
-                  unawaited(context.navigator.pushReplacement(
-                      TopLevelPageRoute(builder: (_) => SignedInScreen())));
-                }
-              },
-              initialUrl: services.get<AppConfig>().webUrl('login'),
-            ),
-          ),
+              constraints: BoxConstraints(
+                maxHeight: mediaQuery.size.height -
+                    400 -
+                    mediaQuery.padding.bottom -
+                    mediaQuery.padding.top,
+              ),
+              child: PrimaryButton(
+                  onPressed: () => browser.open(
+                      url: services.get<AppConfig>().webUrl('login'),
+                      options: InAppBrowserClassOptions()),
+                  child: Text(s.signIn_form_signIn))),
```

Comment:
Please add trailing commas in multi-line widget creations. But now that we open the browser in a separate page, do we really need this `ConstrainedBox`?